src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/ModifyStateAfterAdditionalFeeReceivedCallbackHandler.java

```diff
@@ -129,11 +130,17 @@ private CallbackResponse changeGADetailsStatusInParent(CallbackParams callbackPa
     }
 
     private String getDashboardScenario(CaseData caseData) {
-        if (caseData.getParentClaimantIsApplicant() == YesOrNo.YES && caseData.getIsGaRespondentOneLip() == YES) {
-            return SCENARIO_AAA6_GENERAL_APPLICATION_CREATED_DEFENDANT.getScenario();
+
```

Comment:
These changes suppose to be added getDashboardNotificationScenarioForApplicant() in this method. As getDashboardScenario() is return scenarios for Dashboard Task List